File: Python/Exercicio15.py
"""
A Secretaria de Meio Ambiente que controla o índice de poluição, mantém 3 grupos de indústrias que são
altamente poluentes do meio ambiente. O índice de poluição aceitável varia de 0,05 até 0,25. 
#grupo1
Se o índice sobe para 0,3 as indústrias do 1 grupo são intimadas a suspenderem suas atividades. 
#grupo 2
Se o índice crescer para 0,4 as indústrias do 1 e 2 grupos são intimadas a suspenderem
suas atividades. 
#grupo 3
Se o índice atingir 0,5 todos os grupos devem ser notificados
para paralisarem suas atividades. 
Faça um algoritmo que leia o índice de poluição
medido e emita a notificação adequada aos diferentes grupos de empresas.
"""

indice = float(input("Informe o indice de poluição: "))

if indice < 0.25:
    print("Indice de poluição aceitavel!")
elif indice >= 0.3 and indice < 0.4:
    print("Industrias do Grupo 1 devem suspender suas atividades!")
elif indice >= 0.4 and indice < 0.5:
    print("Industrias dos Grupos 1 e 2 devem suspender suas atividades!")
else:
    print("Todos os grupos devem suspender suas atividades!")

File: C/Exercicio15.c
/*
A Secretaria de Meio Ambiente que controla o indice de poluicao, mantem 3 grupos de industrias que sao
altamente poluentes do meio ambiente. O indice de poluicao aceitavel varia de 0,05 ate 0,25. 
#grupo1
Se o indice sobe para 0,3 as industrias do 1 grupo sao intimadas a suspenderem suas atividades. 
#grupo 2
Se o indice crescer para 0,4 as indutrias do 1 e 2 grupos sao intimadas a suspenderem
suas atividades. 
#grupo 3
Se o indice atingir 0,5 todos os grupos devem ser notificados
para paralisarem suas atividades. 
Faca um algoritmo que leia o indice de poluicao
medido e emita a notificacao adequada aos diferentes grupos de empresas.
*/

#include<stdio.h>

int main(){
	
	float indice;
	
	printf("Informe o indice de poluicao:");
	scanf("%f", &indice);
	
	if(indice >= 0.05 && indice <= 0.25){
		printf("Nenhuma empresa precisa suspender suas atividades!");
	}else if(indice > 0.25 && indice <= 0.3){
		printf("Empresas do grupo 1 devem suspender suas atividades!");
	}else if(indice > 0.3 && indice <= 0.4){
		printf("Empresas do grupo 1 e 2 devem suspender suas atividades!");
	}else{
		printf("Todas as empresas devem suspender suas atividades!");
	}
}
